From 890b0bb69aa6370bce2064f413de4ea07d2bcc26 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Fri, 10 Aug 2018 09:14:01 +0530
Subject: [PATCH] [REVERT ME] Remove bluetooth from feature list

As bluetooth is not yet enabled, having bluetooth feature enabled
results in "Bluetooth keeps stopping" dialog shown in screen
which makes the device unusable.

Fix the issue by removing bluetooth feature from tablet feature
list until bluetooth is enabled.

Change-Id: Ie88df1459654970ddb3e5c04312fbe67eac26270
Tracked-On: OAM-76759
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 data/etc/tablet_core_hardware.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/etc/tablet_core_hardware.xml b/data/etc/tablet_core_hardware.xml
index 6ab8eae..e084470 100644
--- a/data/etc/tablet_core_hardware.xml
+++ b/data/etc/tablet_core_hardware.xml
@@ -31,7 +31,6 @@
     <feature name="android.hardware.location.network" />
     <!-- feature name="android.hardware.sensor.compass" -->
     <!-- feature name="android.hardware.sensor.accelerometer" -->
-    <feature name="android.hardware.bluetooth" />
     <feature name="android.hardware.touchscreen" />
     <feature name="android.hardware.touchscreen.multitouch" />
     <feature name="android.hardware.touchscreen.multitouch.distinct" />
-- 
2.17.1

